### ISSUE TITLE
Make tables focusable and give them a "bypass" button so a keyboard user doesn't have to tab through them

### ISSUE THREAD

**Issue by seblington** [mdn/content#41292]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Temporal

### What specific section or headline is this issue about?

all sections containing tables

### What information was incorrect, unhelpful, or incomplete?

When navigating by keyboard, it was tedious to have to tab through every link in the tables about Temporal namespaces, methods, etc. 

### What did you expect to see?

It would be nice to have the table itself by keyboard focusable (tabindex=0) so that it's possible to focus it and then use arrow keys to scroll. (This is possible currently by focusing one of the links in the table, so that's great that the table is already scrollable).

Add in before each table a hidden "skip link" that skips past all the links in the table and focuses some element after the table, so that keyboard only users can skip past tables they aren't interested in easily without tabbing through them.

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/javascript/reference/global_objects/temporal`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Temporal
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/javascript/reference/global_objects/temporal/index.md
* Last commit: https://github.com/mdn/content/commit/791d6b6a04b1b8c09bc291ebadbc0f99a57dd5f7
* Document last modified: 2025-08-14T03:56:45.000Z

</details>
